include/pokedex.h

```diff
@@ -8,6 +8,7 @@ void ResetPokedex(void);
 u16 GetNationalPokedexCount(u8 caseID);
 u16 GetHoennPokedexCount(u8 caseID);
 u8 DisplayCaughtMonDexPage(u16 species, bool32 isShiny, u32 personality);
+u32 Pokedex_CreateCatchedMonSprite(u32 species, s32 x, s32 y);
```

Comment:
```suggestion
u32 Pokedex_CreateCaughtMonSprite(u32 species, s32 x, s32 y);
```
Sorry :) this throughout the commit

Reply:
OOF, please squash this lmao